<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimerAepWidget</class>
 <widget class="QWidget" name="TimerAepWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>330</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>340</width>
    <height>330</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>340</width>
    <height>330</height>
   </size>
  </property>
  <property name="font">
   <font>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Таймер АЭП</string>
  </property>
  <widget class="QLabel" name="label_Timer_AEP_Preliminarily">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>230</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Предварительное время</string>
   </property>
  </widget>
  <widget class="QLabel" name="m_status_l">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>290</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_previewTime_le">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>250</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="m_start_pb">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>СТАРТ</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="m_editTime_dte">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QTreeWidget" name="m_loadSets_tw">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>231</width>
     <height>190</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string notr="true">Каталоги:</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="m_clear_pb">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Очистить</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Time_Complect">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>210</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Перерыв между комплектами</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_timeSet_le">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>51</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="m_back_pb">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>140</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../path.qrc">
     <normaloff>:/image/button_back.png</normaloff>:/image/button_back.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../path.qrc"/>
 </resources>
 <connections/>
</ui>
